pkg/e2e/operators/cloudingress/certificate.go

```diff
@@ -0,0 +1,70 @@
+package cloudingress
+
+import (
+	"context"
+	"log"
+	"time"
+
+	"github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/openshift/osde2e/pkg/common/constants"
+	"github.com/openshift/osde2e/pkg/common/helper"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+var _ = ginkgo.Describe(constants.SuiteInforming+TestPrefix, func() {
+	h := helper.New()
+	var originalCert string
+	ginkgo.Context("publishingstrategy-certificate", func() {
+		ginkgo.It("IngressController should be patched when Certificate is updated", func() {
+			log.Print("Updating Certificate")
+			ingress1, _ := getingressController(h, "default")
```

Comment:
What is `ingress1` signifying here? The original, or the result? I think this var could be named a bit more clear